apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.ThroughputTool/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.IO;
+
+namespace Google.Cloud.Storage.V1.ThroughputTool
+{
+    public static class Program
+    {
+        private static void Main(string[] args)
+        {
+            Configuration configuration = null;
+            try
+            {
+                configuration = Configuration.FromCommandLineArguments(args);
+                CreateFile(configuration);
+                RunUpload(configuration);
+                RunDownload(configuration);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+            }
+            finally
+            {
+                if (configuration != null)
+                {
+                    File.Delete(configuration.LocalFile);
+                }
+            }
+        }
+
+        private static void CreateFile(Configuration configuration)
+        {
+            var random = new Random();
+            long bytesLeft = configuration.SizeBytes;
+            byte[] buffer = new byte[1024 * 1024];
+            using (var output = File.Create(configuration.LocalFile))
+            {
+                while (bytesLeft > 0)
+                {
+                    random.NextBytes(buffer);
+                    int count = Math.Min((int) bytesLeft, buffer.Length);
```

Comment:
Do the Min as `long`s, then cast to int; otherwise I suspect there could be buuuuuuuugs...

Reply:
Ooh, yes, you're right.